helm/ovn-kubernetes/README.md.gotmpl

```diff
@@ -0,0 +1,98 @@
+{{ template "chart.header" . }}
```

Comment:
I am ok for this to live here for now, once we get the docs website up it would be nice to put this under "installation" section not "helm" section